src/trace_processor/perfetto_sql/stdlib/linux/cpu/utilization/sched.sql

```diff
@@ -13,15 +13,22 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 
--- Computes CPU utilization metric based on the 'sched' table.
-CREATE PERFETTO TABLE cpu_sched_time AS
-SELECT
+-- This table aggregates the total scheduling duration for each CPU.
```

Comment:
This shouldn't be in the utilization package. Please move it one layer up.

Also I would call it something like sched_breakdown or similar. to avoid it being too generic.